HotelApi/packages.config

```diff
@@ -0,0 +1,52 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.7" targetFramework="net472" />
+  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.7" targetFramework="net472" />
+  <package id="Microsoft.AspNet.WebApi.Owin" version="5.2.7" targetFramework="net472" />
+  <package id="Microsoft.AspNet.WebApi.OwinSelfHost" version="5.2.7" targetFramework="net472" />
+  <package id="Microsoft.AspNetCore.Authentication.Abstractions" version="2.2.0" targetFramework="net472" />
```

Comment:
Just to note

For me it looks like you have used default project and didn't optimize your dependecy in the project. A lot of them currently are not used. 
Usually we try to keep only these which are currently under the use.

Reply:
applied